app/views/layouts/application.html.erb

```diff
@@ -102,6 +102,9 @@
     .leaflet-popup-content {
       width: 800px !important;
     }
+    .peoplecard{
```

Comment:
Could you add a space here. It makes it more readable :smile:

Reply:
Sure